common/djangoapps/startup_configurations/validate_config.py

```diff
@@ -0,0 +1,43 @@
+"""
+Common Functions to Validate Configurations
+"""
+from django.conf import settings
+
+
+def validate_lms_config():
+    """
+    Validates configurations for lms and raise ValueError if not valid
+    """
+    validate_common_config()
+
+    # validate feature based configurations
+    validate_marketing_site_config()
```

Comment:
I would recommend making all the functions in here take an explicit settings argument, rather than importing from django.conf.  It will make them easier to test.  The *real* settings can be passed in from `lms/startup.py` and `cms/startup.py`, where it is already imported.